client/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "jest"
```

Comment:
I see that you have added jest for testing but we don't have any tests yet. Would be nice if you could for example create a new test for the component you modified. Call it `MultiStepForm.test.tsx` so jest picks it up.

Reply:
I tried to use jest for some basic testing and added a file `MultiStepForm.test.tsx`, but I was running into issues where  `MultiStepForm.tsx` returns an object but the corresponding test file only expects a JSX since `MultiStepForm.tsx` is a React component. I couldn't get the test to compile so I just removed it for now, and I can remove jest too since we also have Cypress -- but let me know if you would like me to find ways to go through with jest!

Reply:
Ok, you can remove jest for now. I prefer vitest since we are using vite for our build script so you can ignore for now.